installation/io.github.joseexposito.Touchegg.service

```
@@ -0,0 +1,5 @@
+[D-BUS Service]
+Name=io.github.joseexposito.Touchegg
+Exec=/usr/bin/touchegg --daemon
```

```suggestion
Exec=/bin/false
```
This will call the SystemdService

---

Thank for looking into this :smile: I tried that and there seems to be 2 major issues:
- Installation from deb package hangs until the service times out, even thought the service is running in the background
- The service stars even when no clients are connected and crashes every 2 minutes:
```
Feb 20 12:14:57 elementary-os systemd[1]: touchegg.service: start operation timed out. Terminating.
Feb 20 12:14:58 elementary-os systemd[1]: touchegg.service: Failed with result 'timeout'.
Feb 20 12:14:58 elementary-os systemd[1]: Failed to start Touchégg Daemon.
```

It seems like the service is not waiting for the creation of the D-Bus interface even with `/bin/false` :thinking: